runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/MusicClue.java

```diff
@@ -100,7 +157,7 @@ public static MusicClue forText(String text)
 		if (m.find())
 		{
 			final String song = m.group(1);
```

Comment:
@Nightfirecat So basically if we hit this line, the song exists. We could then look for an existing MusicClue using that song, and if it doesn't exist, then just create a new MusicClue?